pom.xml

```diff
@@ -46,4 +46,47 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <!-- profile for nexus-staging:release invocation -->
+      <id>release-staging-repository</id>
+      <activation>
+        <property>
+          <!-- The root project not using nexus-staging-maven-plugin when signing -->
+          <name>!gpg.executable</name>
```

Comment:
Memo: We need to activate this profile in `mvn nexus-staging:release`  https://github.com/googleapis/java-cloud-bom/blob/main/.kokoro/release/stage.sh#L42.